addons/l10n_es_pos/tests/test_frontend.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import odoo.tests
+from odoo.addons.point_of_sale.tests.test_frontend import TestPointOfSaleHttpCommon
+
+
+@odoo.tests.tagged('post_install_l10n', 'post_install', '-at_install')
+class TestUi(TestPointOfSaleHttpCommon):
+    @classmethod
+    def _get_main_company(cls):
+        cls.company_data["company"].country_id = cls.env.ref("base.es").id
+        return cls.company_data["company"]
+
+    def test_spanish_pos(self):
+        simp = self.env['account.journal'].create({
+            'name': 'Simplified Invoice Journal',
+            'type': 'sale',
+            'company_id': self._get_main_company().id,
+            'code': 'SIMP',
+        })
+        get_number_of_regular_invoices = lambda: self.env['account.move'].search_count([('journal_id', '=', self.main_pos_config.invoice_journal_id.id), ('l10n_es_is_simplified', '=', False)])
+        initial_number_of_regular_invoices = get_number_of_regular_invoices()
+        self.main_pos_config.l10n_es_simplified_invoice_journal_id = simp
+        self.main_pos_config.l10n_es_simplified_invoice_limit = 1000
```

Comment:
Put a note above this line that this is linked to the const value (`SIMPLIFIED_INVOICE_LIMIT`) in the tour definition.